sites/fast-component-explorer/webpack.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = (env, args) => {
         devtool: isProduction ? "none" : "inline-source-map",
         entry: {
             main: path.resolve(appDir, "index.tsx"),
-            serviceWorker: path.resolve(appDir, "service-worker-registration.ts"),
+            // serviceWorker: path.resolve(appDir, "service-worker-registration.ts"),
```

Comment:
This should either have a todo task linking to a github item or be removed all together.